backends/custom_cpu/runtime/runtime.cc

```diff
@@ -118,7 +120,9 @@ C_Status Allocate(const C_Device device, void **ptr, size_t size) {
 }
 
 C_Status Deallocate(const C_Device device, void *ptr, size_t size) {
-  free(ptr);
+
+  std::cout << "Dealloc size=" << size << std::endl;	
+      	free(ptr);
```

Comment:
This should be debug code that missed to delete? please avoid change codes of custom cpu.